qa/automation/CareerPortal.Tests.Scenarios/4. Sending CV/4.1.Sending CV.feature

```diff
@@ -0,0 +1,132 @@
+Feature: 4.1. Sending CV
+
+@Type:Positive
+@Priority:Critical
+Scenario Outline: 4.1_01 Correct display of the cv submit form
+    Given user wanted to send CV
+    When user opens "<name of the page>" the page
+    Then user finds the form for sending CV
+	And the form includes "Name", "Email", "Message" fields
+	And the form includes "Attach CV" and "Send" buttons
+	And the form includes checkbox "I agree to the personal data processing"
+
+    Examples: 
+		| name of the page                                               |
+		| https://career.quantori.com/ru                                 |
+		| https://career.quantori.com/ru/positions/cloud-python-engineer |
+
+@Type:Positive
+@Priority:Critical
+Scenario Outline: 4.1_02 Successful submission with filled "Name", "Email" fields, checkbox and attached CV
+    Given user founded the Send CV form on page "<name of the page>"
+    And user entered "<Name>", "<Email>", "<Attach CV>", "<Checkbox>" fields with foolowing combination of values:
+        | Name               | Email               | Attach CV   | Checkbox |
+        | Evgeniia T         | tikeeva_es@mail.ru  | cv.pdf 10Mb | True     |
+        | Евгения Т          | tikeeva1998@mail.ru | cv.doc 2Mb  | True     |
+        | Evgeniia T S       | tikeeva_es@mail.ru  | cv.docx 2Mb | True     |
+        | Ж'еня-Тикеева, мл. | tikeeva1998@mail.ru | cv.rtf 2Mb  | True     |
+	When user clicks "Send" button
+    Then user should see the message that CV was submitted
+
+    Examples: 
+		| name of the page                                               |
+		| https://career.quantori.com/ru                                 |
+		| https://career.quantori.com/ru/positions/cloud-python-engineer |
+
+@Type:Positive
+@Priority:Critical
+Scenario: 4.1_03 Successful submission with filled "Name", "Email", "Message" fields, checkbox and attached CV
+    Given user entered "<Name>", "<Email>", "<Message>", "<Attach CV>", "<Checkbox>" fields with foolowing combination of values:
+        | Name               | Email               | Message                    | Attach CV   | Checkbox |
+        | Evgeniia T         | tikeeva_es@mail.ru  | Please consider my resume. | cv.pdf 10Mb | True     |
+        | Евгения Т          | tikeeva1998@mail.ru | I will be a good worker.   | cv.doc 2Mb  | True     |
+        | Evgeniia T S       | tikeeva_es@mail.ru  | Please consider my resume. | cv.docx 2Mb | True     |
+        | Ж'еня-Тикеева, мл. | tikeeva1998@mail.ru | I will be a good worker.   | cv.rtf 2Mb  | True     |
+	When user clicks "Send" button
+    Then user should see the message that CV was submitted
```

Comment:
Scenario 4.1_02 dublicatates of 4.1_03. I would be choose 4.1_03  but with following edits :
In 39 row after word "values" I would be delete ":"
You missed key word "Example" before you table of values ( see attach)
Also for max coverage I would be add cv file with size 9,9 Mb to test that our form will be send
![111](https://user-images.githubusercontent.com/94995338/144245903-89fd9af3-4b48-48d6-b079-155a2122f260.JPG)

Reply:
Maybe it is correct rename Scenario to Scenario Outline..

Reply:
1. You're right, the grouping makes sense in this case, thanks. I will fix it.
2. After values you need ":" and don't need "Example", since I'm using a table.
![image](https://user-images.githubusercontent.com/95170810/144252410-655f17ef-d591-4da6-a643-197fb3a93f03.png)
3. We have a 10MB boundary condition, so to cover all successful tests we need values less than 10MB (example 2Mb) and 10MB.

Reply:
The same for others scenarious where you have values

Reply:
Also for best coverege yuu can add value "empty" in "Message" to show that CV form will be send

Reply:
Ok, you're right) I'll fix it)